extensions/idp/pkg/backends/cs3/identifier/user.go

```diff
@@ -0,0 +1,67 @@
+package cs3
+
+import (
+	cs3user "github.com/cs3org/go-cs3apis/cs3/identity/user/v1beta1"
+	konnect "github.com/libregraph/lico"
+)
+
+type cs3User struct {
+	u *cs3user.User
+}
+
+func newCS3User(u *cs3user.User) (*cs3User, error) {
+	return &cs3User{
+		u: u,
+	}, nil
+}
+
+func (u *cs3User) Subject() string {
+	return u.u.GetId().GetOpaqueId()
+}
+
+func (u *cs3User) Email() string {
+	return u.u.GetMail()
+}
+
+func (u *cs3User) EmailVerified() bool {
+	return u.u.GetMailVerified()
+}
+
+func (u *cs3User) Name() string {
+	return u.u.GetDisplayName()
+}
+
+func (u *cs3User) FamilyName() string {
+	return ""
+}
+
+func (u *cs3User) GivenName() string {
+	return ""
+}
+
+func (u *cs3User) Username() string {
+	return u.u.GetUsername()
+}
+
+func (u *cs3User) ID() int64 {
+	return u.u.GetUidNumber()
```

Comment:
Hm, this seems wrong (especially knowing how we treat the uidNumber in the user backend by default). And IIUC the `ID()` method is optional. Could we just leave it out?

Reply:
for now ... ok